lib/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/usecase/delegate_load_data_usecase.dart

```diff
@@ -1,14 +1,27 @@
 import 'package:async/async.dart';
 import 'package:seeds/datasource/local/settings_storage.dart';
 import 'package:seeds/datasource/remote/api/eos_repository.dart';
+import 'package:seeds/datasource/remote/api/members_repository.dart';
 import 'package:seeds/datasource/remote/api/proposals_repository.dart';
+import 'package:seeds/datasource/remote/model/delegate_model.dart';
 
 class DelegateLoadDataUseCase {
   final ProposalsRepository _proposalsRepository = ProposalsRepository();
+  final MembersRepository _membersRepository = MembersRepository();
 
-  Future<Result> run() {
+  Future<Result?> run() async {
     final account = settingsStorage.accountName;
+    final result = await _proposalsRepository.getDelegate(account, EosRepository.voiceScopeCampaign);
 
-    return _proposalsRepository.getDelegate(account, EosRepository.voiceScopeCampaign);
+    if (result.isError) {
```

Comment:
Why don't we just return an error result, rather than null?

Reply:
null != error